x-pack/plugins/security_solution/common/endpoint/service/trusted_apps/to_update_trusted_app.ts

```diff
@@ -5,25 +5,27 @@
  * 2.0.
  */
 
-import { MaybeImmutable, NewTrustedApp } from '../../../../../common/endpoint/types';
-import { defaultNewTrustedApp } from '../store/builders';
+import { MaybeImmutable, NewTrustedApp, UpdateTrustedApp } from '../../types';
 
-const NEW_TRUSTED_APP_KEYS: Array<keyof NewTrustedApp> = [
+const NEW_TRUSTED_APP_KEYS: Array<keyof UpdateTrustedApp> = [
   'name',
   'effectScope',
   'entries',
   'description',
   'os',
+  'version',
 ];
 
-export const toNewTrustedApp = <T extends NewTrustedApp>(
+export const toUpdateTrustedApp = <T extends NewTrustedApp>(
   trustedApp: MaybeImmutable<T>
-): NewTrustedApp => {
-  const newTrustedApp = defaultNewTrustedApp();
+): UpdateTrustedApp => {
+  // @ts-ignore
```

Comment:
It's ugly either way. I just tried a cast instead and that seems to also work, so it will look like this:

```
const trustedAppForUpdate: UpdateTrustedApp = {} as UpdateTrustedApp;
```